static/docs/user-guide/external-data/index.md

```diff
@@ -0,0 +1,22 @@
+# External Data
+
+External data are located outside the project directory, either on the local
+filesystem or on a remote host. They may be used in order to:
```

Comment:
remote host sounds like a machine while it can be actually a cloud

Reply:
> remote host sounds like a machine while it can be actually a cloud

"remote host" (like a machine) is one the cases. I am also adding "cloud data storage", like this:

> External data are located outside the project directory, either on the local
filesystem, or on a remote host, or on a cloud data storage.